### PR TITLE
Release notes for v1.0.0-M11

### PR DESCRIPTION
**Pull request by Duhemm**

### CODE REVIEW

File: notes/v1.0.0-M11.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+Bloop now includes a [Mill integration][#532], which means that your build definition can be exported from
+Mill's format to Bloop's.
+
+To try the Mill integration, add the following to the `build.sc` of a project:
+
+```scala
+import $ivy.`ch.epfl.scala::mill-bloop:1.0.0-M11`
+```
+
+And then run `mill bloop.integrations.mill.Bloop/install` from the directory of the mill project.
```
Comment (olafurpg) [inline notes/v1.0.0-M11.md]: Could be `mill.Bloop/install` instead if you move the class to another package?

File: notes/v1.0.0-M11.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+
+# Highlights :books:
+
+## [Redesign the configuration format][#529] (by [Jorge Vicente Cantero][@jvican])
+
+With Bloop 1.0 being around the corner, we need to fix all the issues with the current configuration
+file format, and come up with a solution that will be stable for a long time.
+
+Jorge has been working towards a definitive design for what the configuration file should look like,
+and his intensive head scratching resulted in this pull request.
```
Comment (propensive) [inline notes/v1.0.0-M11.md]: Missing link?

File: notes/v1.0.0-M11.md [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+Using Scala Native or Scala.js with Bloop is completely transparent. If your project uses any of
+these platforms, Bloop will automatically configure itself to link and run using the appropriate
+tools and runtime.
+
+At the moment, running the test is not supported and waiting for [a new release of Scala
+Native][native-pr-1234].
+
+## [Export sbt build information][#538] (by [Jorge Vicente Cantero][@jvican])
+
+When sbt is launched with the sbt-bloop plugin, it will now export immediately Bloop configuration
```
Comment (propensive) [inline notes/v1.0.0-M11.md]: s/export immediately/immediately export/

File: notes/v1.0.0-M11.md [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+The Bloop configuration file now contains paths to the documentation and sources of the dependencies
+of a build. This allows a BSP client to query them from Bloop, and lets IDEs offer a better user
+experience.
+
+## Features :building_construction:
+
+#### [Print test reports][#499] (by [Tim Nieradzik][@tindzk])
+
+Bloop now print a full report of test execution after the command `test` finishes. This gives you
```
Comment (propensive) [inline notes/v1.0.0-M11.md]: s/print/prints/

File: notes/v1.0.0-M11.md [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+Bloop supports an option called `--config-dir`, which defines where Bloop should look for your
+configuration files (the default value is `.bloop` in your current working directory).
+
+When this option was passed, Bloop was creating an absolute path from the base directory where the
+Bloop server had been started, instead of doing so from your current working directory. This patch
+fixes this incorrect behavior!
+
+## Usability improvements
+
+#### [Improve error reporter][#537] (by [Gabriele Petronella][@gabro])
```
Comment (propensive) [inline notes/v1.0.0-M11.md]: s/Improve/Improved/

File: notes/v1.0.0-M11.md (hunk 1 above)
Comment (Duhemm) [reply notes/v1.0.0-M11.md]: I imagine so, I admit I have no experience with mill... These instructions come from #532 

PR welcome :)

**Comment by Duhemm** [comment]
Thanks a lot @propensive !

File: notes/v1.0.0-M11.md (hunk 1 above)
Comment (olafurpg) [reply notes/v1.0.0-M11.md]: I would consider it if I used mill. Cc/ @baccata

Comment (Baccata) [reply notes/v1.0.0-M11.md]: I wanted to be consistent with how the sbt integration is packaged. Personally I just have a top level task that delegate to that module so I just have to call `mill bloop`, so I'm not really annoyed by that.

**Comment by Baccata** [comment]
Thanks a lot for the hard work guys 😄

**Comment by jvican** [comment]
184 commits since the last release! It was about time :smile:

**Comment by siddhartha-gadgil** [comment]
I was trying this and got the following error:
```
Failed to resolve ivy dependencies:
  ch.epfl.scala:mill-bloop_2.12:1.0.0-M11 
    not found: /home/gadgil/.ivy2/local/ch.epfl.scala/mill-bloop_2.12/1.0.0-M11/ivys/ivy.xml
    not found: https://repo1.maven.org/maven2/ch/epfl/scala/mill-bloop_2.12/1.0.0-M11/mill-bloop_2.12-1.0.0-M11.pom
```

I had in my `build.sc`:
```
import $ivy.`ch.epfl.scala::mill-bloop:1.0.0-M11`
```

Should I be adding a repository?

**Comment by Duhemm** [comment]
@siddhartha-gadgil Thanks for the report. I released it this morning, but something may have gone wrong. I'll check that now.

**Comment by Duhemm** [comment]
@siddhartha-gadgil good catch! I forgot to release the staging repository. That's the risk of doing things manually 😄
Here's the pom, on Sonatype: https://oss.sonatype.org/service/local/repositories/releases/content/ch/epfl/scala/mill-bloop_2.12/1.0.0-M11/mill-bloop_2.12-1.0.0-M11.pom

It looks from your error message that everything should work in a short time, it's just a matter of waiting until the artefacts are synced to the Central repository. You can probably add the Sonatype Release repository to your build if you don't want to wait, but I don't know Mill well enough to tell you how to do that.

**Comment by siddhartha-gadgil** [comment]
Thanks. I'll wait for the sync.

**Pull request merged**